src/run_environment/reference_genome.ml

```diff
@@ -144,11 +144,9 @@ module Default = struct
   let b38 =
     (* Release 79 *)
     let b38_url =
-      "ftp://ftp.ensembl.org/pub/release-79/fasta/homo_sapiens/dna/\
-       Homo_sapiens.GRCh38.dna.primary_assembly.fa.gz" in
+      "ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/hg38/hg38bundle/Homo_sapiens_assembly38.fasta.gz" in
     let dbsnp_b38 =
-      "http://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b142_GRCh38/\
-       VCF/00-All.vcf.gz" in
+      "ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/hg38/hg38bundle/dbsnp_144.hg38.vcf.gz" in
     let gtf_b38_url =
       "http://ftp.ensembl.org/pub/release-79/gtf/homo_sapiens/\
        Homo_sapiens.GRCh38.79.gtf.gz" in
```

Comment:
Yea that makes sense. I was hoping to avoid that since we will end up with a split - no hg38 GTF, can't find a b38 dbSNP etc.